src/nvim/eval.lua

```diff
@@ -123,16 +123,16 @@ return {
     feedkeys={args={1, 2}, base=1},
     file_readable={args=1, base=1, func='f_filereadable'},  -- obsolete
     filereadable={args=1, base=1},
```

Comment:
Maybe add `filereadable` as well while you're at it :)

Reply:
Done